packages/spectacle/src/components/slide-layout.tsx

```diff
@@ -82,14 +88,127 @@ const List = ({
   );
 };
 
+/**
+ * Generic vertically-centered Header layout
+ */
+const Header = ({
+  flexBoxProps,
+  headingProps,
+  heading,
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  heading: string;
+  flexBoxProps?: ComponentProps<typeof FlexBox>;
+  headingProps?: ComponentProps<typeof Heading>;
+}) => (
+  <Slide {...rest}>
+    <FlexBox height="100%" {...flexBoxProps}>
+      <Heading {...headingProps}>{heading}</Heading>
+    </FlexBox>
+  </Slide>
+);
+
+/**
+ * Section layout with left aligned text
+ */
+const Section = ({
+  sectionTitle,
+  sectionTitleProps,
+  ...rest
+}: Omit<SlideProps, 'children'> & {
+  sectionTitle: string;
```

Comment:
Same here, I think it'd be nice for this to be `string | ReactNode`, just to add a little bit of flexibility.

Reply:
I hadn't considered this. I think it's a good idea in both place. Thanks!

Reply:
@gksander, rather than type the `sectionTitle` as `string | ReactNode`, why not just allow `children` through instead of `Omit`-ing?

Reply:
Changed my mind, I don't like my last idea; passing `children` through is not the same as allowing the heading content  to have types other than `string`. 

Question though -
Would it be more succinct to type the section title as a `ReactNode`, like `sectionTitle: ReactNode`, since `ReactNode` is a union that includes `string`?

Reply:
Oh no, I'm just seeing this comment after leaving my last review!

Great point about `string | ReactNode` being superfluous – should just be `ReactNode`! Although now I'm a bit torn on using `children` vs another prop. Maybe we can have a quick chat about this today and align on a direction?